gril/grilreply.c

```diff
@@ -894,8 +916,13 @@ GSList *g_ril_reply_parse_get_calls(GRil *gril, const struct ril_msg *message)
 		}
 
 		parcel_r_int32(&rilp); /* namePresentation */
-		parcel_r_int32(&rilp); /* uusInfo */
-
+		uusInfoPresent = parcel_r_int32(&rilp); /* uusInfo */
+		if (uusInfoPresent == 1) {
+			parcel_r_int32(&rilp); /* Type */
+			parcel_r_int32(&rilp); /* Dcs */
+			//Read byte array with user data and ignore it		
+			parcel_r_raw(&rilp, &arrayLen);
```

Comment:
`parcel_r_raw()` returns a `g_try_malloc0()` pointer. Please don't forget to free it using `g_free()`.